CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Not released
 
+- Add `Authorization header` fetching tiles [#519](https://github.com/CartoDB/carto-react/pull/519)
```

Comment:
You always have to describe it in terms of Fix | Add | Remove xxxx feature.... so Add is not 100% clear. It's better to use

```suggestion
- Fix TileLayer unauthorized error due to [breaking changes](https://github.com/visgl/deck.gl/pull/7287) in deck.gl >=8.9.0-alpha.3 and >=8.8.15 [#519](https://github.com/CartoDB/carto-react/pull/519)
```